src/components/Login/index.jsx

```diff
@@ -68,33 +71,35 @@ const Login = () => {
       auth, loginData.email, loginData.password
     ).then((response) => {
       console.log("Successfully Loged In.", response)
+      setCurrentUserAuthId(response.user.uid)
       nav("/show_expense");
     }).catch((error) => {
+      alert(error.message)
       console.log("Error: While Login.", error)
     })
   }
 
   return (
     <div className="main-head">
-      <div className="main">  	
+      <div className="main">  
         <input type="checkbox" id="chk" aria-hidden="true" />
           <div className="signup">
-            <form>
-              <label htmlFor="chk" aria-hidden="true">Sign up</label>
-              <input type="text" name="name" placeholder="Name" required="" />
-              <input type="text" name="phone" placeholder="Phone" required="" />
-              <input type="text" name="username" placeholder="User name" required="" />
-              <input type="email" name="email" placeholder="Email" required="" />
-              <input type="password" name="pswd" placeholder="Password" required="" />
-              <button>Sign up</button>
+            <form onSubmit={ createUserToFirebaseDB }>
+            <label htmlFor="chk" aria-hidden="true">Sign up</label>
+              <input onChange={ changeHandlerData } value={ data.name } type="text" name="name" placeholder="Name" required="" />
+              <input onChange={ changeHandlerData } value={ data.phone } type="text" name="phone" placeholder="Phone" required="" />
+              <input onChange={ changeHandlerData } value={ data.username } type="text" name="username" placeholder="User name" required="" />
+              <input onChange={ changeHandlerData } value={ data.email } type="email" name="email" placeholder="Email" required="" />
+              <input onChange={ changeHandlerData } value={ data.password } type="password" name="password" placeholder="Password" required="" />
```

Comment:
I think there is a library for validating the field in react.
For Refernce [Yep](https://www.npmjs.com/package/yup)